server/src/main/java/com/broadcom/lsp/cobol/service/CobolLanguageServer.java

```diff
@@ -126,12 +134,45 @@ private void getLocaleFromClient() {
 
   @Override
   public CompletableFuture<Object> shutdown() {
-    return supplyAsync(() -> TRUE);
+    LOG.info("LS for COBOL received shutdown request");
```

Comment:
Should be "COBOL LS received shutdown request"

Reply:
done